src/components/TodoList.js

```diff
@@ -75,7 +75,10 @@ function TodoList() {
                   <img src={check} alt="" />
                 ) : null}
               </div>
-              {item.title}
+              <p>{item.title}</p>
```

Comment:
I dont think this is semantic

could there be a better tag/element to use instead of `<p>`?

Reply:
where would we place/show date of completion?

I think there could be a little not too tiny bar under each todo that could show date of creation and date of completion and perhaps other similar details

let me know which arrangement you want to go with befoe implementation

well done

Reply:
something like the position where the due date  is here

<img width="308" alt="image" src="https://github.com/Highshirt99/TODOTODAY/assets/53216981/d8271b84-67e5-4676-809e-58f5535154d2">